editoast/src/views/temporary_speed_limits.rs

```diff
@@ -0,0 +1,216 @@
+use axum::extract::Json;
+use axum::extract::State;
+use axum::Extension;
+use chrono::NaiveDateTime;
+use derivative::Derivative;
+use editoast_schemas::infra::DirectionalTrackRange;
+use serde::de::Error as SerdeError;
+use serde::{Deserialize, Serialize};
+use std::result::Result as StdResult;
+use utoipa::ToSchema;
+
+use crate::error::Result;
+use crate::models::temporary_speed_limits::TemporarySpeedLimit;
+use crate::models::Changeset;
+use crate::views::AuthorizationError;
+use crate::views::AuthorizerExt;
+use crate::CreateBatch;
+use crate::Model;
+use editoast_authz::BuiltinRole;
+
+use crate::AppState;
+
+crate::routes! {
+    "/temporary_speed_limits" => create,
+}
+
+editoast_common::schemas! {
+    TemporarySpeedLimitCreateForm,
+    TemporarySpeedLimitCreateResponse,
+    TemporarySpeedLimitItemForm,
+}
+
+#[derive(Serialize, Derivative, ToSchema)]
```

Comment:
It doesn't seem to be used so better remove it.

```suggestion
#[derive(Serialize, ToSchema)]
```